save-frontend/src/main/kotlin/org/cqfn/save/frontend/App.kt

```diff
@@ -135,7 +135,7 @@ class App : RComponent<PropsWithChildren, AppState>() {
                         }
                         Route {
                             attrs {
-                                path = arrayOf("/:owner/:name/history/execution/:executionId/details/:testSuiteName/:pluginName/:testFilePath")
+                                path = arrayOf("/:owner/:name/history/execution/:executionId/details/:testSuiteName/:pluginName/:testFilePath+")
```

Comment:
Why + not *?

Reply:
There should be at least file name, so at least one part in the repeated group